src/components/review/Review.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import './Review.css';
+
+export default function Review({ data }) {
+  const { avatars, description, subtitle, title } = data;
+
+  const [displayedReview, setReview] = React.useState(description);
+  const [isReviewExpanded, setExpanded] = React.useState(false);
+
+  const lengthLimit = 290;
+  const shortenedReview = `${description.slice(0, lengthLimit)}..."`;
```

Comment:
This is genius! I implemented this with css in a different project with something like a multiline overflow ellipsis, but I didn't even consider limiting the string itself 🤯

Reply:
💯